NDTensors/src/lib/UnallocatedArrays/src/abstractfill/set_types.jl

```diff
@@ -42,34 +73,9 @@ SetParameters.default_parameter(::Type{<:AbstractFill}, ::Position{3}) = Tuple{}
 
 SetParameters.nparameters(::Type{<:AbstractFill}) = Val(3)
 
-# Set parameter 1
-## Right now using AbstractArray
-## TODO These are more difficult because T is technically defined so need some way to strip T of it {} types
-# function set_parameter(
-#   T::Type{<:AbstractFill{<:Any,<:Any,P3}}, ::Position{1}, P1
-# ) where {P3}
-#   return T{P1,<:Any,P3}
-# end
-# function set_parameter(
-#   T::Type{<:AbstractFill{<:Any,P2,P3}}, ::Position{1}, P1
-# ) where {P2,P3}
-#   return T{P1,P2,P3}
-# end
-# function set_parameter(
-#   T::Type{<:AbstractFill{<:Any,<:Any,P3}}, ::Position{2}, P2
-# ) where {P3}
-#   return T{<:Any,P2,P3}
-# end
-# function set_parameter(
-#   T::Type{<:AbstractFill{P1,<:Any,P3}}, ::Position{2}, P2
-# ) where {P1,P3}
-#   return T
-# end
-# set_parameter(T::Type{<:AbstractFill}, ::Position{3}, P3) = T{<:Any,<:Any,P3}
-# set_parameter(T::Type{<:AbstractFill{P1}}, ::Position{3}, P3) where {P1} = T{<:Any,P3}
-# function set_parameter(T::Type{<:AbstractFill{<:Any,P2}}, ::Position{3}, P3) where {P2}
-#   return T{<:Any,P2,P3}
-# end
-
-# Set parameter 2
-## using AbstractArray
+## This is a helper function which can be renamed.
+## Essentially, because I use AbstractFill I don't know what the name
+## of the object is in `set_parameter`, so I use this to find the 
+## name with out any of the parameters attached.
```

Comment:
@mtfishman Thank you for reviewing these changes! I should have been more clear, this was an end of the day push from my workstation so that I could pick up the work easily on my laptop remotely and this comment specifically was a note to myself which I was going to post on Github as a comment/question to you. I'm still working through your suggestion so the code is not yet ready to review again but I do appreciate your looking through it! Especially the `set_eltype` functions. I was trying to determine if I could write a generic version that didn't depend on `Zeros` or `Fill`

Reply:
I removed this comment and wrote a hopefully more comprehensive comment.